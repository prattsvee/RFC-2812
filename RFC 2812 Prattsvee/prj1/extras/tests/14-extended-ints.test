# multiple int's with internal underscores

123_456_789

1_2_3
