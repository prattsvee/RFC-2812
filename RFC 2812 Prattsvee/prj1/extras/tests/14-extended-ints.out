[
  {
    "%k": "int",
    "%v": 123456789
  },
  {
    "%k": "int",
    "%v": 123
  }
]
